R/repl.R

```diff
@@ -37,6 +37,31 @@
 #'
 #' @seealso [py], for accessing objects created using the Python REPL.
 #'
+#' @section Magics: A handful of magics are supported in `repl_python()`:
+#'
+#' Lines prefixed with `!` are executed as system commands:
+#'  - `!cmd --arg1 --arg2`: Execute arbitrary system commands
+#'
+#' Magics start with a `%` prefix. Supported magics include:
+#'  - `%conda ...` executes a conda command in the active conda environment
+#'  - `%pip ...` executes pip for the active python.
+#'  - `%load`, `%loadpy`, `%run` executes a python file.
+#'  - `%system`, `!!` executes a system command and capture output
+#'  - `%env`: read current environment variables.
+#'    - `%env name`: read environment variable 'name'.
+#'    - `%env name=val`, `%env name val`: set environment variable 'name' to 'val'.
+#'       `val` elements in `{}` are interpolated using f-strings (required Python >= 3.6).
+#'  - `%cd <dir>` change working directory.
+#'     - `%cd -`: change to previous working directory.
+#'     - `%cd -3`: change to 3rd most recent working directory.
```

Comment:
Do these operations work regardless of how the working directory was changed? Or is it only specifically for directory changes done via `%cd`? (I suspect the latter; if so, I think that's worth documenting here.)